content/php/concepts/date-functions/terms/date-sub/date-sub.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'date_sub()'
+Description: 'Subtracts a specified interval from a given date.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Time'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`date_sub()`** function subtracts a number of days, months, years, hours, minutes and seconds from a given date.
+
+## Syntax
+
+```pseudo
+date_sub($object, $interval)
+```
+
+The `date_sub()` function has two required parameters:
+
+- `$object`: This parameter specifies the DateTime object returned by date_create().
```

Comment:
```suggestion
- `$object`: This parameter specifies the `DateTime` object returned by `date_create()`.
```

Reply:
generally we use backticks on specific terms, functions, symbols specific to the code shown or language. If it's a generic term such as arrays or classes I would omit the backticks.